loopchain/peer/block_manager.py

```diff
@@ -140,12 +142,28 @@ def _send_unconfirmed_block(self, block_: Block, target_reps_hash, round_: int):
             f"target_reps_hash({target_reps_hash})")
 
         block_dumped = self.blockchain.block_dumps(block_)
-        ObjectManager().channel_service.broadcast_scheduler.schedule_broadcast(
+        send_kwargs = {"block": block_dumped, "round_": round_, "channel": self.__channel_name}
+
+        release_recovery_mode = False
+        if conf.RECOVERY_MODE:
+            from loopchain.tools.recovery import Recovery
+            if self.blockchain.block_height <= Recovery.release_block_height():
+                util.logger.info(f"broadcast block({block_.header.height}) from recovery node")
+                send_kwargs.update({"from_recovery": True})
+
+            if self.blockchain.block_height >= Recovery.release_block_height():
```

Comment:
Is that right to set True to release_recovery_mode at Recovery.release_block_height()?

Reply:
Condition is correct.